velox/exec/tests/OperatorUtilsTest.cpp

```diff
@@ -529,6 +532,47 @@ TEST_F(OperatorUtilsTest, projectChildren) {
           srcRowVector->childAt(projection.inputChannel).get());
     }
   }
+
+  {
+    // Test wrapping an unloaded lazy vector in dictionary vector multiple
```

Comment:
This method appears to be very long. Would it be possible to put the new test in a separate method?

Reply:
Added a test `TEST_F(OperatorUtilsTest, projectDuplicateChildren)`.